test/benchmarks/driver_bench/src/suites/multi_bench/large_doc_bulk_insert.mts

```diff
@@ -0,0 +1,30 @@
+import { driver, type mongodb } from '../../driver.mjs';
+
+export const taskSize = 27.31;
+
+let collection: mongodb.Collection;
+let documents: any[];
+let largeDoc: any;
+
+export async function before() {
+  largeDoc = await driver.load('single_and_multi_document/large_doc.json', 'json');
+}
+
+export async function beforeEach() {
+  await driver.drop();
+  await driver.create();
+
+  // Make new "documents" so the _id field is not carried over from the last run
```

Comment:
I think this was a bug in the old tests, pretty sure they were using the same array but dropped the collection so we weren't making new ObjectIds after the first test which is a requirement in the spec.